es8/2017-07/jul-25.md

```diff
@@ -0,0 +1,547 @@
+
+MF: I showed this to demonstrate that there are no implementation difficulties.
+
+AWB: The purpose of stage 3 is to get feedback from implementation and usage.
+
+BT: I would like to see two full-green columns before stage 4, though I'm curious why Edge is not passing more of the tests. We haven't gotten any bugs as a result of implementing this spec, though.
+
+PHE: Source code typically doesn't make it onto embedded devices (although neither does the parser).
+
+MF: I think the benefits of having F.p.toString standardized still remain.
+
+PST: Why would it be a problem to make returning the source code optional? Couldn't you return a string that gives a syntax error? Like `function () { [native code ] }`?
+
+MF: Forward compatibility: can't produce something now that is guaranteed to be a syntax error in the future.
+
+MF: I will bring this proposal back later, when we have alleviated concerns about web compatibility.
+
+AWB: The XS embedded JavaScript implementors are providing legitimate feedback by saying they won't/can't implement this feature.
+
+AK: This proposal has an issue tracker where we can continue detailed discussion.
+
+PHE: Content Security Policies can forbid compiling/eval'ing strings, and this is kind of the inverse: the environment might declare that it can't decompile functions.
+
+JHD: If you're not going to implement the feature at all, what does it matter what the spec for it says?
+
+PST: I would rather it was configurable than for XS to refuse to implement something required.
+
+PHE: I would have no problem moving the sentence that requires F.p.toString to implement this proposal to Annex B (web compatibility).
+
+
+#### Conclusion/Resolution
+
+- Wait for intentional implementation feedback, not just compatibility data.
+- Make sure to talk with XS implementors, on the issue tracker for this proposal.
+
+
+
+## 11.ii.b. Symbol.prototype.description 
+
+(Michael Ficarra)
+
+- [proposal](https://michaelficarra.github.io/Symbol-description-proposal/)
+
+MF: Proposal to put a "description" getter on Symbol.prototype that returns `desc` given a symbol created with `Symbol(desc)`
+
+YK: The most common reason is to writing another debugging output. They don't necessary want to write Symbol(...).
+
+??: Its documentation, why woulnd't you want it?
+
+WH: There are at least two ways in the existing spec to get at the description:
+
+- `Symbol.prototype.toString`
+- *SetFunctionName*
+
+...They differ in their treatment of undefined descriptions. `Symbol.prototype.toString` treats it as though it were an empty description, while *SetFunctionName* distinguishes it. Do we want to distinguish between undefined and empty string?
+
+JHD: Just tried to set the name of a function. It doesn't distinguish between undefined and empty symbol descriptions.
```

Comment:
this (and the following 2 lines) should probably be removed; it turns out that it differentiates only when using a Symbol as a concise method name.